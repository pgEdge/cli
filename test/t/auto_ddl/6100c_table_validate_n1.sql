
-- Final AutoDDL validation for the 6100 series on n1 to ensure all the DROP TABLE performed in the 6100b files on n2 
-- was auto replicated to n1.
-- None of the Tables should exist and spock.tables should not contain any entries for these tables

-- Prepared statement for spock.tables so that we can execute it frequently in the script below
PREPARE spocktab AS SELECT nspname, relname, set_name FROM spock.tables WHERE relname = $1 ORDER BY relid;

-- Final validation of all tables along with querying the spock.tables
-- validating all tables dropped on n1
\d+ employees
EXECUTE spocktab('employees');

\d+ departments
execute spocktab('departments');

\d+ projects
execute spocktab('projects');

\d+ employee_projects
execute spocktab('employee_projects');

\d+ products
execute spocktab('products');

\d+ "CaseSensitiveTable"
execute spocktab('CaseSensitiveTable');

\d+ test_tab1
execute spocktab('test_tab1');

\d+ test_tab2
execute spocktab('test_tab2');

\d+ test_tab3
execute spocktab('test_tab3');

\d+ test_tab4
EXECUTE spocktab('test_tab4');

\d+ test_tab5
EXECUTE spocktab('test_tab5');
