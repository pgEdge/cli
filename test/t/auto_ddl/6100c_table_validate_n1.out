-- Final AutoDDL validation for the 6100 series on n1 to ensure all the DROP TABLE performed in the 6100b files on n2 
-- was auto replicated to n1.
-- None of the Tables should exist and spock.tables should not contain any entries for these tables
-- Prepared statement for spock.tables so that we can execute it frequently in the script below
PREPARE spocktab AS SELECT nspname, relname, set_name FROM spock.tables WHERE relname = $1 ORDER BY relid;
PREPARE
-- Final validation of all tables along with querying the spock.tables
-- validating all tables dropped on n1
\d+ employees
Did not find any relation named "employees".
EXECUTE spocktab('employees');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ departments
Did not find any relation named "departments".
execute spocktab('departments');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ projects
Did not find any relation named "projects".
execute spocktab('projects');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ employee_projects
Did not find any relation named "employee_projects".
execute spocktab('employee_projects');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ products
Did not find any relation named "products".
execute spocktab('products');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ "CaseSensitiveTable"
Did not find any relation named ""CaseSensitiveTable"".
execute spocktab('CaseSensitiveTable');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ test_tab1
Did not find any relation named "test_tab1".
execute spocktab('test_tab1');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ test_tab2
Did not find any relation named "test_tab2".
execute spocktab('test_tab2');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ test_tab3
Did not find any relation named "test_tab3".
execute spocktab('test_tab3');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ test_tab4
Did not find any relation named "test_tab4".
EXECUTE spocktab('test_tab4');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ test_tab5
EXECUTE spocktab('test_tab5');
